liquibase.properties

```diff
@@ -0,0 +1,7 @@
+# This file is used by the liquibase plugin to perform manual migrations/rollbacks
+classpath=src/main/resources
+changeLogFile=src/main/resources/db/changelog/db.changelog-master.yaml
+url=jdbc:mysql://localhost:49694/ems_integration_application
```

Comment:
This is specific to local, the port code varies to individual machine, please add to gitignore, so that we can have a local copy of this file.

Reply:
done